account/src/lib.rs

```diff
@@ -160,7 +161,7 @@ fn authenticate(
         }
         env.crypto().ed25519_verify(
             &signature.public_key,
-            &signature_payload.clone().into(),
+            &signature_payload.to_bytes().into(),
```

Comment:
This change is because `Hash<>`, that replaced `BytesN<>` for this use case, does not implement `Clone`. Fixing that in:
- https://github.com/stellar/rs-soroban-sdk/pull/1302